capz/templates/ci/patches/kubeadm-bootstrap-windows-ci.yaml

```diff
@@ -24,7 +24,7 @@
       ctr.exe -n k8s.io images tag docker.io/sigwindowstools/kube-proxy:v1.23.1-calico-hostprocess "docker.io/sigwindowstools/kube-proxy:${CI_VERSION/+/_}-calico-hostprocess"
 
       kubeadm.exe version -o=short
-      kubectl.exe version --client=true --short=true
+      kubectl.exe version --client=true
```

Comment:
should we add a `echo "kubectl version: "` or something similar to these statements while we are here?

Reply:
yea, i'll do that!

Reply:
updated!